api/data/NMBS/liveboard.php

```diff
@@ -114,22 +114,21 @@ private static function parseData($xml, $time, $lang, $fast = false)
             $journey = $data->Journey[$i];
 
             $left = 0;
+            $canceled = false;
             $delay = (string) $journey['delay'];
             if ($delay == '-') {
                 $delay = '0';
             }
-
-            $platform = '';
-            if (isset($journey['platform'])) {
-                $platform = (string) $journey['platform'];
+            
+            if ($delay == 'cancel') {
+                $delay = 999999; // Indicate something is wrong if `canceled` is not read by the client
```

Comment:
Should be 0 though... 27 hours is just random ;)

Maybe just making it empty? It's not relevant anymore?